man/man8/zpool.8

```diff
@@ -1760,6 +1763,9 @@ pool, in addition to the pool-wide statistics.
 Omit statistics since boot.
 Normally the first line of output reports the statistics since boot.
 This option suppresses that first line of output.
+.It Fl n
+Print headers only once when passed 
+.Ar interval
```

Comment:
Can you move this between the -L and -p options, and what do you think about rephrasing it as "Display the header only once rather than periodically.".

Reply:
Done